### PR TITLE
Updated rebalancing set token with internal audit fixes. Added furthe…

### PR DESCRIPTION
**Pull request by bweick**
…r testing for fee distribution.

### CODE REVIEW

File: contracts/core/RebalancingSetToken.sol [hunk 1]
```diff
@@ -846,23 +862,52 @@ contract RebalancingSetToken is
...
+    /**
+     * Get and instance of the Core contract
+     *
+     * @return  ICore    Instance of Core contract
+     */
+    function getCoreInstance()
+        private
+        returns (ICore)
+    {
+        return ICore(IRebalancingSetFactory(factory).core());
```
Comment (asoong) [inline contracts/core/RebalancingSetToken.sol]: Why does this need to be its own function? This adds gas for no reason

File: contracts/core/RebalancingSetToken.sol [hunk 2]
```diff
@@ -846,23 +862,52 @@ contract RebalancingSetToken is
         return minimumBid.mul(_unit).div(_naturalUnit).div(auctionPriceDivisor);
     }
 
+    /**
+     * Function to calculate the total amount of fees owed
+     *
+     * @param   _quantity       Amount of Sets to take fees from
+     * @param   _managerFee    Fee, in basis points, to be taken from base amount of Sets
```
Comment (asoong) [inline contracts/core/RebalancingSetToken.sol]: Align me

File: test/contracts/core/rebalancingSetToken.spec.ts [hunk 3]
```diff
@@ -1335,6 +1470,7 @@ contract('RebalancingSetToken', accounts => {
       await coreMock.issue.sendTransactionAsync(rebalancingSetToken.address, rebalancingSetQuantityToIssue);
 
       subjectCaller = managerAccount;
+      console.log(rebalanceFee);
```
Comment (asoong) [inline test/contracts/core/rebalancingSetToken.spec.ts]: Remove

Comment (felix2feng) [inline test/contracts/core/rebalancingSetToken.spec.ts]: Remove me

File: test/contracts/core/rebalancingSetToken.spec.ts [hunk 4]
```diff
@@ -1296,8 +1432,8 @@ contract('RebalancingSetToken', accounts => {
     let subjectCaller: Address;
 
     let proposalPeriod: BigNumber;
-    let entranceFee: BigNumber;
-    let rebalanceFee: BigNumber;
+    let entranceFee: BigNumber = ZERO;
+    let rebalanceFee: BigNumber = undefined;
```
Comment (felix2feng) [inline test/contracts/core/rebalancingSetToken.spec.ts]: Isn't this implicit?

**Pull request merged**